packages/ui/src/views/layouts/PullRequestLayout.tsx

```diff
@@ -21,105 +36,70 @@ enum PullRequestTabsKeys {
   CHANGES = 'changes'
 }
 
-const PullRequestLayout: FC<PullRequestLayoutProps> = ({
+export const PullRequestLayout: FC<PullRequestLayoutProps> = ({
   usePullRequestStore,
   useTranslationStore,
   spaceId,
   repoId,
   updateTitle
 }) => {
-  const { NavLink, Outlet } = useRouterContext()
+  const { Outlet } = useRouterContext()
   const { pullRequest } = usePullRequestStore()
   const { t } = useTranslationStore()
+  const navigate = useNavigate()
+  const location = useLocation()
+
+  const getIsActiveTab = useCallback(
+    (tab: string) => (location.pathname.endsWith(tab) ? 'active' : 'inactive'),
+    [location.pathname]
+  )
+  const makeHandleTabChange = useCallback((tab: string) => () => navigate(tab), [navigate])
+
+  const getTabProps = useCallback(
```

Comment:
Using a `useCallback` here won't make any positive difference to performance. `useCallback` is great for stabilising function references when passing a function to another component/hook etc. In this case it's only being used within the component and called on every render anyway. Perhaps a better idea would be to have a `useMemo` for each unique instance instead to make sure they aren't regenerated on each render.

Reply:
Got rid of useCallback